test/test_download_tests.py

```diff
@@ -58,7 +58,7 @@ def test_redownload_test_files_multiple_times_works(self):
 
         download_tests(self.tmpPath)
 
-    @pytest.mark.single
+    @pytest.mark.unit
```

Comment:
Just to make sure I understand, this isn't actually related to this PR, right? (The convention in ADS is to use the term `unit`, but it looks like this was missed from some previous PR or something like that, right?)

Reply:
Yep - thought I'd squeeze this in here since I noticed it